share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
Fair enough on the security issue. The summit last week would disagree that not everything can be made reproducible but I'll stay away as it's not important here :wink:.



> That we should support marjor need as default, but keep custom behavior friendly which will only apply to minority people and come with lower support.

Why does it need to be documented in the manpage?